### ISSUE TITLE
figure out with of the 4 SR is optimal for a given signal

### ISSUE THREAD

**Issue by kratsg** [kratsg/Optimization#41]
@swiatlow: let's talk about signal regions now?

Max Swiatlowski [11:22 AM] 
Cool

Max Swiatlowski [11:22 AM]
Did you see my slides?

Giordon Stark [11:23 AM] 
the new ones?

Giordon Stark [11:23 AM]
with the 4 signal regions?

Max Swiatlowski [11:23 AM] 
Yeah

Giordon Stark [11:23 AM] 
I don't understand the 4 particularly.

Max Swiatlowski [11:23 AM] 
4?

Giordon Stark [11:23 AM] 
do you mean 2 signal regions for low lumi, and 2 for high lumi?

Max Swiatlowski [11:24 AM] 
Yup

Max Swiatlowski [11:24 AM]
Low boost/ high boost

Max Swiatlowski [11:24 AM]
It's very approximate

Max Swiatlowski [11:24 AM]
The first thing to do is just implement these 4 SRs

Max Swiatlowski [11:25 AM]
Then check, for each lumi, which SR is best for each point

Max Swiatlowski [11:25 AM]
And what the significance is

Giordon Stark [11:25 AM] 
so a SR == a cut?

Max Swiatlowski [11:25 AM] 
If the sig is very different from the OPTIMAL you found... We might need to adjust, or add a new SR, etc.

Max Swiatlowski [11:25 AM]
SR is a set of cuts

Max Swiatlowski [11:26 AM]
It's the table on slide 15 or whatever

Giordon Stark [11:26 AM] 
err, a SR = a supercut?

Max Swiatlowski [11:26 AM] 
Yeah with all pivot

Max Swiatlowski [11:26 AM]
Just fixed cuts

Giordon Stark [11:26 AM] 
gotcha. No problem.

Max Swiatlowski [11:26 AM] 
Then, you check which Sr is best at each point, etc etx

Giordon Stark [11:26 AM] 
so I run all 4 supercut's I have

Max Swiatlowski [11:27 AM] 
Yeah

Giordon Stark [11:27 AM] 
and just look at all the significances reported for a given signal region

Giordon Stark [11:27 AM]
find which of the 4 maximizes the signal region

Giordon Stark [11:27 AM]
and just make a grid showing 1,2,3,4

Giordon Stark [11:27 AM]
basically saying which one maximized that box?

Max Swiatlowski [11:27 AM] 
Yup!

**Comment by kratsg** [comment]
initially started with https://github.com/kratsg/Optimization/commit/1c2a28d6f25cdc27acf905777cab0e5edeafe7b4
